utils/set-alert-policy-required-datasources.ts

```diff
@@ -6,11 +6,8 @@ import {
   isNotRemoved,
 } from './lib/github-api-helpers';
 import Quickstart from './lib/Quickstart';
-import Alert, {
-  AlertPolicySetRequiredDataSourcesMutationResults,
-  SubmitSetRequiredDataSourcesMutationResult,
-} from './lib/Alert';
-import { chunk } from './lib/nr-graphql-helpers';
+import Alert, { SubmitSetRequiredDataSourcesMutationResult } from './lib/Alert';
```

Comment:
I assume `AlertPolicySetRequiredDataSourcesMutationResults` was an unused import?

Reply:
looking at the file i do not see it being used